experiments/AtmosLES/risingbubble.jl

```diff
@@ -214,8 +214,7 @@ function config_risingbubble(FT, N, resolution, xmax, ymax, zmax)
     # The model coefficient for the turbulence closure is defined via the [CLIMAParameters
     # package](https://climate-machine.github.io/CLIMAParameters.jl/latest/)
     # A reference state for the linearisation step is also defined.
-    ref_state =
-        HydrostaticState(DryAdiabaticProfile(typemin(FT), FT(300)), FT(0))
+    ref_state = HydrostaticState(DryAdiabaticProfile(FT(300)), FT(0))
```

Comment:
Can't we omit the second argument `FT(0)`?

Reply:
Yep, I'll make an outer constructor that defaults to this.